spec/api-web-request-spec.js

```diff
@@ -80,19 +85,19 @@ describe('webRequest module', () => {
 
     it('receives details object', (done) => {
       ses.webRequest.onBeforeRequest((details, callback) => {
-        assert.strictEqual(typeof details.id, 'number')
-        assert.strictEqual(typeof details.timestamp, 'number')
-        assert.strictEqual(typeof details.webContentsId, 'number')
-        assert.strictEqual(details.url, defaultURL)
-        assert.strictEqual(details.method, 'GET')
-        assert.strictEqual(details.resourceType, 'xhr')
-        assert(!details.uploadData)
+        expect(details.id).to.be.a('number')
+        expect(details.timestamp).to.be.a('number')
+        expect(details.webContentsId).to.be.a('number')
+        expect(details.url).to.equal(defaultURL)
```

Comment:
this can also check for property type i'd say!
`expect(details.method).to.be.a('string).that.is.equal('GET')`